src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -180,6 +180,9 @@
     /** what class to use for mbean registeration */
     MBEAN_REGISTRATION_CLASS("org.apache.cassandra.mbean_registration_class"),
 
+    /** Which class to use for token metadata provider */
+    CUSTOM_TMD_PROVIDER_PROPERTY("cassandra.custom_token_metadata_provider_class"),
```

Comment:
could we please follow `org.apache.cassandra...` convention for these properties?

Reply:
This is a good request and @jacek-lewandowski made a similar one for STAR-842 and I created STAR-869 as a follow-up. Since these config options are currently embedded into CNDB we should change them everywhere after we've finished all of our porting work so as not to introduce config mismatches or other headaches. I'll open a ticket.